### ISSUE TITLE
[Psr7] Installing `symfony/psr-http-message-bridge` does not work on 5.4

### ISSUE THREAD

**Issue by artyuum** [symfony/symfony-docs#19018]
I tried installing `symfony/psr-http-message-bridge` on a new Symfony 5.4 project exactly how the documentation showed (`composer require symfony/psr-http-message-bridge`) but it didn't work and I had the following error:
>   Problem 1
    - Root composer.json requires symfony/psr-http-message-bridge 5.4, found symfony/psr-http-message-bridge[v0.1, v0.2, v0.3, v0.4, v1.0.0, ..., v1.3.0, v2.0.0, ..., 2.3.x-dev, 6.4.x-dev, 7.0.x-dev] but it does not match the constraint.

After having a talk with @stof on Slack, the following command worked:
```bash
composer require symfony/psr-http-message-bridge:2.3
```

I'm not sure how to correctly word that out in a PR so I'll open an issue to point out this issue instead.

**Comment by javiereguiluz** [comment]
I know nothing about this 😐 Is the `2.3` constraint mandatory only for Symfony 5.4 or also for some of the upper maintained branches (6.3, 6.4 and 7.0)? Thanks.

**Comment by derrabus** [comment]
This is a bug that we don't have a good solution for, unfortunately.

This is a workaround for Symfony 5.4 and 6.3:

```sh
composer require symfony/psr-http-message-bridge:^2.3
```

Note that I've added a `^` in order to not pin the bridge to version 2.3.0, but to allow 2.3.1 and potential newer 2.3.x releases as well.

The issue should be gone in Symfony 6.4 and 7.0 because the bridge will be released in a version 6.4.0 and 7.0.0.

**Comment by javiereguiluz** [comment]
Perfectly explained. Thanks Alexander!
